### PR TITLE
LoginFormAuthenticator example updated

### PR DESCRIPTION
**Pull request by carlcasbolt**
RouterInterface has been replaced with UrlGeneratorInterface

### CODE REVIEW

File: security/form_login_setup.rst [hunk 1]
```diff
@@ -171,10 +171,10 @@ a traditional HTML form that submits to ``/login``:
 
     use App\Entity\User;
     use Doctrine\ORM\EntityManagerInterface;
-
+
```
Comment (skmedix) [inline security/form_login_setup.rst]:
```suggestion

```

File: security/form_login_setup.rst [hunk 2]
```diff
@@ -253,13 +254,14 @@ a traditional HTML form that submits to ``/login``:
                 return new RedirectResponse($targetPath);
             }
 
-            // For example : return new RedirectResponse($this->router->generate('some_route'));
+            // For example : return new RedirectResponse($this->urlGenerator->generate('some_route'));
             throw new \Exception('TODO: provide a valid redirect inside '.__FILE__);
+
```
Comment (skmedix) [inline security/form_login_setup.rst]: Please remove extra line

**Review by skmedix — CHANGES_REQUESTED** [review]

File: security/form_login_setup.rst (hunk 1 above)
Comment (carlcasbolt) [reply security/form_login_setup.rst]: fixed

File: security/form_login_setup.rst (hunk 2 above)
Comment (carlcasbolt) [reply security/form_login_setup.rst]: fixed

**Comment by javiereguiluz** [comment]
The proposed changes are correct according to https://github.com/symfony/maker-bundle/blob/master/src/Resources/skeleton/authenticator/LoginFormAuthenticator.tpl.php. Thanks  @carlcasbolt.

**Pull request merged**